packages/remix-server-runtime/cookies.ts

```diff
@@ -1,12 +1,16 @@
-import type { CookieParseOptions, CookieSerializeOptions } from "cookie";
+import type {
+  CookieParseOptions as RRCookieParseOptions,
+  CookieSerializeOptions as RRCookieSerializeOptions,
+} from "cookie";
 import { parse, serialize } from "cookie";
 
 import type { SignFunction, UnsignFunction } from "./crypto";
 import { warnOnce } from "./warnings";
 
-export type { CookieParseOptions, CookieSerializeOptions };
+export type CookieParseOptions = RRCookieParseOptions;
+export type CookieSerializeOptions = RRCookieSerializeOptions;
```

Comment:
These are still interfaces in RR, so re-exporting them as type aliases

Once RR exports these as types, we can go back to

```
export type { CookieParseOptions, CookieSerializeOptions };
```